htslib/vcf.h

```diff
@@ -205,6 +205,10 @@ typedef struct bcf_dec_t {
 #define BCF_ERR_CTG_INVALID   32
 #define BCF_ERR_TAG_INVALID   64
 
+// get the error descriptions
+HTSLIB_EXPORT
+const char* bcf_strerror(int errorcode, char* buffer, size_t maxbuffer);
```

Comment:
For consistency with other code nearby, pointer declarations should be in the C style rather than the C++ style:

```suggestion
const char *bcf_strerror(int errorcode, char *buffer, size_t maxbuffer);
```

Reply:
I've always wondered where that meme came from.

It never made sense to me, and indeed it's positively dangerous.  To a naive reader the spacing implies that `char* a,b;` is a pair of char pointers, which is plainly not true.  Glad to know I can also blame it on C++ now :-)

Reply:
The style came hand in hand with another C++ style rule that says “never declare more than one variable in the same declaration” — which avoids the danger (provided you follow it).

Reply:
updated pointer declaration to c style